firm_constraint:
    lower:
        - -225.80
        - -1.97
        - -2.68
        - -2.12
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0
    upper:
        - 225.80
        - 1.30
        - 2.68
        - 2.12
        - 0.0
        - 0.0
        - 0.0
        - 0.0
        - 0.0
#END
